packages/esm-patient-search-app/src/compact-patient-search/compact-patient-banner.component.tsx

```diff
@@ -91,9 +92,16 @@ const PatientSearchResults = React.forwardRef<HTMLDivElement, PatientSearchResul
                 />
               </div>
               <div>
-                <h2 className={styles.patientName}>{`${patient.name?.[0]?.given?.join(' ')} ${
-                  patient.name?.[0]?.family
-                }`}</h2>
+                <div className={styles.flexRow}>
+                  <h2 className={styles.patientName}>{`${patient.name?.[0]?.given?.join(' ')} ${
+                    patient.name?.[0]?.family
+                  }`}</h2>
+                  <ExtensionSlot
+                    extensionSlotName="patient-banner-tags-slot"
+                    state={{ patient }}
+                    className={styles.flexRow}
+                  />
+                </div>
```

Comment:
Hey @jwnasambu , we don't have to show all the extensions in here but only the deceased one.

This will pull every tag from the patient.

The following should work:
```
<ExtensionSlot
    extensionSlotName="patient-banner-tags-slot"
    state={{ patient }}
    className={styles.flexRow}
    select={(extensions) => extensions.filter(ext => ext.name === "deceased-patient-tag")}
  />
```

Please run and test this one out.
Thanks!

Reply:
@vasharma05 I have tested out and it works quite fine bro! Thanks for the review.